src/components/Personalization/createPersonalizationDetails.js

```diff
@@ -103,9 +103,7 @@ export default ({
       return viewCache.isInitialized();
     },
     shouldFetchData() {
-      return (
-        this.hasScopes() || this.hasSurfaces() || !this.isCacheInitialized()
-      );
+      return this.hasScopes() || this.hasSurfaces();
```

Comment:
What is this change for?

Reply:
I'll revert it. It was needed for testing.